meta-lmp-base/recipes-containers/docker/docker-ce/0001-dynbinary-use-go-cross-compiler.patch

```diff
@@ -1,114 +0,0 @@
-From 6c1d9dea0ef2000cdeec38b46d60b2db66160ed3 Mon Sep 17 00:00:00 2001
```

Comment:
Same.

Reply:
have that on https://github.com/foundriesio/meta-lmp/pull/1444. fixed now